apps/asap-server/src/entities/team.ts

```diff
@@ -11,7 +11,9 @@ export const updateSchema = Joi.object({
       }),
     )
     .required(),
-});
+})
+  .min(1)
```

Comment:
Users can remove every tool on the profile, the min should be 0.

Reply:
That would mean that tools is an empty array, no?